src/core/Chip/Chip.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { Component, ReactNode } from 'react';
+import classnames from 'classnames';
+import { default as styled } from 'styled-components';
+import { baseStyles } from './Chip.baseStyles';
+import { HtmlButton, HtmlSpan } from '../../reset';
+import { TokensProp, InternalTokensProp } from 'core/theme';
+import { withSuomifiDefaultProps } from '../theme/utils';
+import { Icon } from '../Icon/Icon';
+import { logger } from '../../utils/logger';
+import { VisuallyHidden } from '../../components/Visually-hidden/Visually-hidden';
+
+const baseClassName = 'fi-chip';
+const disabledClassName = `${baseClassName}--disabled`;
+const iconClassName = `${baseClassName}--icon`;
+const contentClassName = `${baseClassName}--content`;
+const removableClassName = `${baseClassName}--removable`;
+const buttonClassName = `${baseClassName}--button`;
+
+type ChipVariant = 'static' | 'default';
+
+interface InternalChipProps {
+  /** Chip element content */
```

Comment:
should we add styles to allow overriding e.g. max-width without using the attached classnames? Not sure if we should, just wondering.

Reply:
I think that this is a simple enough component that if you really want a chip that wide you can quite easily build your own or do some extra tricks with this one. I don't think it's worthwhile to support that out of the box.